deps/fix_repl.jl

```diff
@@ -0,0 +1,11 @@
+using PackageCompiler
+
+prec_jl_url = "https://raw.githubusercontent.com/JuliaLang/julia/release-1.3/contrib/generate_precompile.jl"
```

Comment:
I think we might have to match this with the version of Julia which is used when installing MKL. Otherwise it could reference functions that don't exist.

Reply:
Thanks! I've pushed in a new commit that does not use the hard-coded version but match the julia version.